configs/app/features/noves.ts

```diff
@@ -0,0 +1,23 @@
+import type { Feature } from './types';
+
+import { getEnvValue } from '../utils';
+
+const novesEnabled = getEnvValue('NEXT_PUBLIC_NOVES_ENABLED') === 'true';
+
+const title = 'Noves API';
+
+const config: Feature<{ isEnabled: true }> = (() => {
```

Comment:
Recently, we introduced our own transaction interpretation service. So, please consider to extend the feature file `configs/app/features/txInterpretation.ts` by adding new value to `NEXT_PUBLIC_TRANSACTION_INTERPRETATION_PROVIDER` env (e.g "noves")